karapace/schema_registry_apis.py

```diff
@@ -309,6 +310,9 @@ async def schemas_get(self, content_type, *, schema_id):
             response_body["schemaType"] = schema.schema_type
         self.r(response_body, content_type)
 
+    async def schemas_types(self, content_type):
+        self.r(["JSON", "AVRO"], content_type)
```

Comment:
It is very easy to get these out-of-sync. It would make more sense to have this as an attribute of:

https://github.com/aiven/karapace/blob/f9f0ef9987e4af4f519c11b6481bcc606cd4f511/karapace/schema_reader.py#L52

Reply:
The issue currently is that Protobuf is unsupported: https://github.com/aiven/karapace/issues/67, thus `TypedSchema` entries don't match the schema types supported. Once Protobuf is supported, we can make this endpoint return the `TypedSchema` entries. The other approach is to make this endpoint lie about the supported endpoints. I wouldn't do that. I added a comment a reminder in #67 to add PROTOBUF to this endpoint results once Protobuf's supported.